doc/AST_FORMAT.md

```diff
@@ -802,6 +802,31 @@ Format:
  ~~~~~~~~~~~~~~~~~ expression
 ~~~
 
+### "Endless" method
+
+Format:
+
+~~~
+(def_e :foo (args) (int 42))
```

Comment:
Not sure about the naming. Any suggestions?

Reply:
I like it. Also could've been called something like "def one-liner", but as I understand you still can use `\` to break into multiple lines, so technically there's no such requirement.

Also, we need to emit some information about the `=` token.

Reply:
I was thinking about these new nodes again, and now I'm not sure we need them at all: what about re-using `def` and `defs`? The absence of the `end` could indicate the endless-ness of the definition.

As well as the presence of the `=` (not sure how to call it, `assignment`?) attribute.

```
~~~
(def :foo (args) (int 42))
"def foo = 42"
 ~~~ keyword
     ~~~ name
         ^ assignment
 ~~~~~~~~~~~~ expression
~~~
```

Reply:
that would break all the code expecting `end` to be non-nullable. I'm sure some libraries don't have `if` checks (because these checks don't make any sense atm), so it'd be a breaking change.

Reply:
Good point. Thanks! 

What about the `assignment` name for the `=` attribute?